openshift-kube-apiserver/admission/customresourcevalidation/node/validate_node_config.go

```diff
@@ -78,18 +97,47 @@ func validateConfigNodeForExtremeLatencyProfile(obj, oldObj *configv1.Node) *fie
 	return nil
 }
 
-func (configNodeV1) ValidateCreate(_ context.Context, uncastObj runtime.Object) field.ErrorList {
+func validateMinimumKubeletVersion(nodesGetter corev1client.NodesGetter, obj *configv1.Node) *field.Error {
```

Comment:
I think we should probably clarify somewhere that this validation on admission is by the nature of a distributed system only a best effort. Inflight operations might still result in older kubelet bypassing it if the right lucky timing takes place.
I'm not heavily involved with this piece of the code base so not sure what would be the usual way to communicate something like that but may be documenting the function is enough // validateMinimumKubeletVersion does a best effort validation...?